files/puppet_helper.groovy

```diff
@@ -546,6 +546,10 @@ class Actions {
               throw new UnsupportedCredentialsClass("unsupported " + keyStoreSource)
           }
           break
+        case 'com.google.jenkins.plugins.credentials.oauth.GoogleRobotPrivateKeyCredentials':
+          info['account_id'] = cred.getServiceAccountConfig().getAccountId()
+          info['private_key'] = new String(cred.getServiceAccountConfig().getPrivateKey().getEncoded())
```

Comment:
Others use `IOUtils.toString`. As someone not that familiar with groovy, is there a difference?

Reply:
TBH I'm not sure either but I'm happy to make that change.

Reply:
Maybe some more experienced Jenkins/groovy user could comment on this.

Reply:
It looks OK as-is.  A literal string is being used to avoid requiring all of those classes to be loaded.